src/tasks/copy-template-files.ts

```diff
@@ -10,134 +10,99 @@ import ncp from "ncp";
 import path from "path";
 import { promisify } from "util";
 import link from "../utils/link";
-import {getArgumentFromExternalExtensionOption} from "../utils/external-extensions";
+import { getArgumentFromExternalExtensionOption } from "../utils/external-extensions";
 
 const EXTERNAL_EXTENSION_TMP_FOLDER = "tmp-external-extension";
 const copy = promisify(ncp);
 let copyOrLink = copy;
 
-
 const expandExtensions = (options: Options): Extension[] => {
   const expandedExtensions = options.extensions
-    .map((extension) => extensionDict[extension])
-    .map((extDescriptor) =>
-      [extDescriptor.extends, extDescriptor.value].filter(isDefined)
-    )
+    .map(extension => extensionDict[extension])
+    .map(extDescriptor => [extDescriptor.extends, extDescriptor.value].filter(isDefined))
     .flat()
     // this reduce just removes duplications
-    .reduce(
-      (exts, ext) => (exts.includes(ext) ? exts : [...exts, ext]),
-      [] as Extension[]
-    );
+    .reduce((exts, ext) => (exts.includes(ext) ? exts : [...exts, ext]), [] as Extension[]);
 
   return expandedExtensions;
 };
 
-const isTemplateRegex = /([^\/\\]*?)\.template\./;
+const isTemplateRegex = /([^/\\]*?)\.template\./;
 const isPackageJsonRegex = /package\.json/;
 const isYarnLockRegex = /yarn\.lock/;
-const isNextGeneratedRegex = /packages\/nextjs\/generated/
-const isConfigRegex = /([^\/\\]*?)\\config\.json/;
-const isArgsRegex = /([^\/\\]*?)\.args\./;
```

Comment:
![Screenshot 2024-05-23 at 5 19 18 PM](https://github.com/scaffold-eth/create-eth/assets/80153681/58d30002-effd-42c7-b2b7-76a0be9bbba4)


Lol, it not best practice to touch working regex and I could have added eslint ignore comment about them 

But removing the backslash made sense checkout [this](https://chatgpt.com/share/eb379c4f-a663-4c87-bae4-8b06bab520c4), also tested it nicely and works nicely